README.md

```diff
@@ -1,21 +1,20 @@
 # Breez Liquid SDK
 
-- `lib`: [lib/README.md](lib/core/README.md)
-- `cli`: [cli/README.md](cli/README.md)
-- Bindings
-  - `python`: [lib/bindings/README.md]()
+- `lib`: [lib/README.md](../../lib/core/README.md)
+- `cli`: [cli/README.md](../../cli/README.md)
+- Bindings: [bindings/README.md](../../lib/bindings/README.md)
```

Comment:
This README is in the root, so doesn't need the `../../`

Reply:
I think you mean't the `packages/dart/README.md` file

Reply:
Yes, it was symlinked..

Reply:
@dangeross
Reverted as part of [Update README of Dart/Flutter packages](https://github.com/breez/breez-liquid-sdk/pull/353/commits/97c4b386cb01ee331ae814def3d0e3d6b35bf0b8).